### PR TITLE
Blinds widget: replace labels with icons and set widget colors consistent with other widgets

### PR DESCRIPTION
**Pull request by davidlb**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -144,6 +144,15 @@
...
       </symbol>
+      <symbol id="arrow-down-drop-circle" viewBox="0 0 24 24">
+        <path d="M12,2A10,10 0 0,1 22,12A10,10 0 0,1 12,22A10,10 0 0,1 2,12A10,10 0 0,1 12,2M7,10L12,15L17,10H7Z" />
+      </symbol>
+      <symbol id="arrow-up-drop-circle" viewBox="0 0 24 24">
+        <path d="M12,22A10,10 0 0,1 2,12A10,10 0 0,1 12,2A10,10 0 0,1 22,12A10,10 0 0,1 12,22M17,14L12,9L7,14H17Z" />
+      </symbol>
+      <symbol id="stop-circle" viewBox="0 0 24 24">
+         <path d="M12,2A10,10 0 0,0 2,12A10,10 0 0,0 12,22A10,10 0 0,0 22,12A10,10 0 0,0 12,2M9,9H15V15H9" />
+      </symbol>
```
Comment (t0mg) [inline public/index.html:155]: Any reason why you didn't go with the simple arrow-up arrow-down and stop ?

**Review by t0mg — COMMENTED** [review]
Yeah I think that's probably for the best and it'll make theming easier. I'm just not entirely sold on the icons yet :)

File: public/index.html (hunk 1 above)
Comment (davidlb) [reply public/index.html:155]: In fact, the stop icon is smaller than arrows. As I prefer to have consistent size in icons, I used these ones. It's up to you to choose the ones you prefer ;)

Comment (t0mg) [reply public/index.html:155]: Good point

**Review by t0mg — APPROVED** [review]
Thanks!

**Comment by t0mg** [comment]
Thanks for the cleanup. Consistency is good.
It would be nice if you could update the wiki accordingly :)

**Comment by davidlb** [comment]
Here is the screenshot for the wiki:

image

**Pull request merged**